docs/integrations/builtin/credentials/postgres.md

```diff
@@ -6,22 +6,64 @@ contentType: integration
 
 - [Postgres](/integrations/builtin/app-nodes/n8n-nodes-base.postgres/)
 - [Agent](/integrations/builtin/cluster-nodes/root-nodes/n8n-nodes-langchain.agent)
 
 /// note | Agent node users
-The Agent node doesn't support tunnels
+The Agent node doesn't support SSH tunnels.
 ///
+
 ## Prerequisites
 
-Create an user account on a Postgres server. 
+[Create a user account](https://www.postgresql.org/docs/current/sql-createuser.html) on a Postgres server. 
+
+## Supported authentication methods
+
+- Database connection
+
+## Related resources
+
+Refer to [Postgres's documentation](https://www.postgresql.org/docs/16/index.html){:target=_blank .external-link} for more information about the service.
+
+## Using database connection
+
+To configure this credential, you'll need:
+
+- The **Host**: Enter the host or domain name for the server.
+- The **Database**: Enter the database name.
+- A **User** name: Enter the username for the user account.
+- A user **Password**: Enter the password for the user account.
+- **Ignore SSL Issues**: When turned on, the credential will connect even if SSL validation fails.
+- **SSL**: Choose whether to use SSL in your connection. Refer to Postgres [SSL Support](https://www.postgresql.org/docs/16/libpq-ssl.html){:target=_blank .external-link} for more information. Options include:
+    - **Allow**: Sets the `ssl-mode` parameter to `allow`. First try a non-SSL connection; if that fails, try an SSL connection.
+    - **Disable**: Sets the `ssl-mode` parameter to `disable`. Only try a non-SSL connection.
+    - **Require**: Sets the `ssl-mode` parameter to `require`. Only try an SSL connection. If a root CA file is present, verify that a trusted certificate authority (CA) issued the server certificate.
+    - **Verify**: Sets the `ssl-mode` parameter to `verify-ca`. Only try an SSL connection and verify that a trusted certificate authority (CA) issued the server certificate.
+    - **Verify-Full**: Sets the `ssl-mode` parameter to `verify-full`. Only try an SSL connection, verify that a trusted certificate authority (CA) issued the server certificate and that the requested server host name matches that in the certificate.
+- The **Port**: Enter the port number to use for the connection.
+- **SSH Tunnel**: If turned on, the credential uses SSH to encrypt the network connection with the Postgres server. Refer to [Using SSH tunnel](#using-ssh-tunnel) for more information.
+
+Refer to [Connection Strings](https://www.postgresql.org/docs/16/libpq-connect.html#LIBPQ-CONNSTRING){:target=_blank .external-link} for more information on finding and formatting these fields.
+
+### SSH tunnel
+
+Only turn on the **SSH Tunnel** if:
 
-## Using Database Connection
+- You're using the credential with the [Postgres](/integrations/builtin/app-nodes/n8n-nodes-base.postgres/) node (Agent node doesn't support SSH tunnels).
+- You have an SSH server running on the same machine as the Postgres server.
+- You can log in using `ssh` as some user.
```

Comment:
What does 'as some user' mean?

Reply:
Ah this was a copy/paste from Postgres that I didn't update well. It really needs to be "You have a user account that can log in using `ssh`.